src/eko/matching_conditions/as2.py

```diff
@@ -268,7 +261,7 @@ def A_hg(n, sx, L):
                 * (n + 1)
                 * (n + 2)
                 * (2 + n + n**2)
-                * (10 * S1m**2 - 9 * Sp2m + 26 * S2m + 9 * Sp2p)
+                * (10 * S1m**2 + 18 * (-1) ** n * (2 * Sm2 + zeta2) + 26 * S2m)
```

Comment:
Error: `(-1)**n` - I believe this can **not** appear in the implementation! I will appear in the mathematical expression, but then gets eliminated upon choosing ns+/ns-

Reply:
true true...